logger.hpp

```diff
@@ -0,0 +1,43 @@
+/* -*- Mode:C++; c-file-style:"gnu"; indent-tabs-mode:nil; -*- */
```

Comment:
Maybe remove this file considering. See how PSync uses the logger.
Just ndn-cxx/logger include at the top and NDN_LOG_INIT(ndnsd.MyClass) to get desired logs.

Reply:
done.